pyModeS/adsb.py

```diff
@@ -18,8 +18,7 @@
 """
 
 import math
-from . import util
-from .util import crc
+from pyModeS import util
```

Comment:
There is no problem for explicit relative imports (the `.`), even in Python 3 it is recommended I think. Especially when we are are using a relative package within the library. 

I though `from pyModeS import ...` will only import from installed package. Right?

Imaging that I have install the latest pyModeS from pip, I add some new functions in `util.py` at master branch, which is not released yet. The absolute import with raise an error when I am trying to run the code. And this problem can not be solved by virtualenv, because new version has not been released yet in pip. 

am I correct or missing anything here?